package/src/components/AutoCompleteInput/AutoCompleteSuggestionItem.tsx

```diff
@@ -55,94 +167,3 @@ const styles = StyleSheet.create({
     paddingHorizontal: 8,
   },
 });
-
-const AutoCompleteSuggestionItemWithContext = ({
-  itemProps,
-  triggerType,
-}: AutoCompleteSuggestionItemPropsWithContext) => {
-  const {
-    theme: {
-      colors: { accent_blue, black, grey },
-      messageInput: {
-        suggestions: {
-          command: { args: argsStyle, container: commandContainer, title },
-          emoji: { container: emojiContainer, text },
-          mention: { avatarSize, column, container: mentionContainer, name: nameStyle },
-        },
-      },
-    },
-  } = useTheme();
-
-  if (triggerType === 'mention') {
-    const { id, image, name, online } = itemProps as SuggestionUser;
-    return (
-      <View style={[styles.container, mentionContainer]}>
-        <Avatar image={image} name={name} online={online} size={avatarSize} />
-        <View style={[styles.column, column]}>
-          <Text style={[styles.name, { color: black }, nameStyle]} testID='mentions-item-name'>
-            {name || id}
-          </Text>
-        </View>
-        <AtMentions pathFill={accent_blue} />
-      </View>
-    );
-  } else if (triggerType === 'emoji') {
-    const { name, unicode } = itemProps as Emoji;
-    return (
-      <View style={[styles.container, emojiContainer]}>
-        <Text style={[styles.text, { color: black }, text]} testID='emojis-item-unicode'>
-          {unicode}
-        </Text>
-        <Text style={[styles.text, { color: black }, text]} testID='emojis-item-name'>
-          {` ${name}`}
-        </Text>
-      </View>
-    );
-  } else if (triggerType === 'command') {
-    const { args, name } = itemProps as SuggestionCommand;
-    return (
-      <View style={[styles.container, commandContainer]}>
-        <AutoCompleteSuggestionCommandIcon name={name} />
-        <Text style={[styles.title, { color: black }, title]} testID='commands-item-title'>
-          {(name || '').replace(/^\w/, (char) => char.toUpperCase())}
-        </Text>
-        <Text style={[styles.args, { color: grey }, argsStyle]} testID='commands-item-args'>
-          {`/${name} ${args}`}
-        </Text>
-      </View>
-    );
-  } else {
-    return null;
-  }
-};
-
-const areEqual = (
```

Comment:
Same question as above, why are we removing the memoization ? Again, not against it - but we have to have very good rationale' to do it otherwise this is how performance issues emerge.

Reply:
Each item will have its own item data, and the trigger type will also be sent directly from the top-level list component. I don't see much of a benefit in adding a memo here, but I am happy to change if you see I am wrong somewhere. And on top of it in the list the component is already wrapped in a `useCallback` which was more important per se.

Reply:
I have a proposal to remove the props from these components and rather use the composer inside it directly to get `queryText` and `triggerType` in that case we will not have to rely on props at all and can move the `renderItem` in the parent component outside making it faster. Wdyt?

Reply:
I have added it back. Will think about these improvements later so that I don't mess things up here.